src/test/java/uk/gov/pay/directdebit/payments/services/GoCardlessServiceTest.java

```diff
@@ -197,8 +192,10 @@ public void confirm_shouldThrow_ifFailingToCreateMandateInGoCardless() {
 
     @Test
     public void confirm_shouldThrow_ifFailingToCreatePaymentInGoCardless() {
+        when(mockedGoCardlessClientWrapper.createCustomer(PAYMENT_REQUEST_EXTERNAL_ID, payer)).thenReturn(goCardlessCustomer);;
+        when(mockedGoCardlessClientWrapper.createCustomerBankAccount(PAYMENT_REQUEST_EXTERNAL_ID, goCardlessCustomer, payer.getName(), payer.getSortCode(), payer.getAccountNumber())).thenReturn(goCardlessCustomer);
+        when(mockedPayerService.getPayerFor(transaction)).thenReturn(payer);
```

Comment:
These three lines appear in several tests: can we move them into an `@Before` method?

Reply:
sure, let me take a look at what makes sense to move to the setup method